tests/integration/relationships/belongs-to-test.js

```diff
@@ -11,10 +15,11 @@ const { attr, hasMany, belongsTo } = DS;
 const { hash } = RSVP;
 
 let env, store, User, Message, Post, Comment, Book, Chapter, Author, NewMessage;
-const injectionValue = Ember.MODEL_FACTORY_INJECTIONS;
 
 module("integration/relationship/belongs_to Belongs-To Relationships", {
   beforeEach() {
+    setupModelFactoryInjections();
```

Comment:
this was not previously setup for all tests, rather some tests explicitly opted in.